pldoc/static/js/colors.json

```diff
@@ -0,0 +1,174 @@
+[{
```

Comment:
Also, this .json file isn't valid JSON - it's an array. JSON has to be an object at the top level. You could do:

```javascript
{
  colors: [
    <contents of this file>
  ]
}
```

and then it'd become valid JSON.